src/pages/Books/Books.tsx

```diff
@@ -1,15 +1,68 @@
+import { useState } from 'react';
+import SearchRoundedIcon from '@mui/icons-material/SearchRounded';
+import clsx from 'clsx';
+
 import { Layout } from '../../components/Layout/Layout';
+import { BookCard } from '../../components/BookCard/BookCard';
+import { CurrentBookCard } from '../../components/CurrentBookCard/CurrentBookCard';
+import mockbooks from '../../data/mockbooks.json';
+import { Book } from '../../../types/types';
 import style from './Books.module.scss';
 
-export const Books = () => (
-  <Layout>
-    <div className={style.pageSection}>
-      <h2 className="heading2">
-        books books books books books books books books books books books books
-        books books books books books books books books books books books books
-        books books books books books books books books books books books books
-        books books books books books books books books books books books books
-      </h2>
-    </div>
-  </Layout>
-);
+export const Books: React.FC = () => {
+  const books: Book[] = mockbooks;
```

Comment:
you can move this outside of the component since it has no references to the inside